array-manipulation-3/src/zip.js

```diff
@@ -0,0 +1,14 @@
+/* exported zip */
+function zip(first, second) {
+  var newArr = [];
+  if (first.length === second.length || first.length < second.length) {
+    for (var i = 0; i < first.length; i++) {
+      newArr.push([first[i], second[i]]);
+    }
+  } else if (first.length > second.length) {
```

Comment:
This can just be an `else`. If the lengths aren't equal or `first` isn't less than `second`, then the length of `second` _is_ greater than `first`